client/lib/wpcom-undocumented/lib/undocumented.js

```diff
@@ -1149,6 +1149,11 @@ Undocumented.prototype.readRecommendationsStart = function( query, fn ) {
 	return this.wpcom.req.get( '/read/recommendations/start', query, fn );
 };
 
+Undocumented.prototype.graduateNewReader = function( fn ) {
+	const params = { apiVersion: '1.2' };
+	return this.wpcom.req.post( '/read/graduate-new-reader', params, fn );
```

Comment:
ah, i think this needs an empty body as the second param. like `this.wpcom.req.post( '/read/graduate-new-reader', {}, params, fn );`